scipy/optimize/_minimize.py

```diff
@@ -1000,27 +1000,33 @@ def _remove_from_bounds(bounds, i_fixed):
     return Bounds(lb, ub)  # don't mutate original Bounds object
 
 
-def _remove_from_func(fun_in, i_fixed, x_fixed, min_dim=None, remove=0):
```

Comment:
Closures like the one I removed don't work with pickling, which is necessary if one wants to use `multiprocessing`.
One solution is to create a callable class.